test/e2e/inspect_test.go

```diff
@@ -132,28 +132,27 @@ var _ = Describe("Podman inspect", func() {
 	})
 
 	It("podman inspect with mount filters", func() {
-		SkipIfRemote()
 
-		ctrSession := podmanTest.Podman([]string{"create", "-v", "/tmp:/test1", ALPINE, "top"})
+		ctrSession := podmanTest.Podman([]string{"create", "--name", "test", "-v", "/tmp:/test1", ALPINE, "top"})
 		ctrSession.WaitWithDefaultTimeout()
 		Expect(ctrSession.ExitCode()).To(Equal(0))
 
-		inspectSource := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Source}}"})
+		inspectSource := podmanTest.Podman([]string{"inspect", "test", "--format", "{{(index .Mounts 0).Source}}"})
 		inspectSource.WaitWithDefaultTimeout()
 		Expect(inspectSource.ExitCode()).To(Equal(0))
 		Expect(inspectSource.OutputToString()).To(Equal("/tmp"))
 
-		inspectSrc := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Src}}"})
+		inspectSrc := podmanTest.Podman([]string{"inspect", "test", "--format", "{{(index .Mounts 0).Src}}"})
 		inspectSrc.WaitWithDefaultTimeout()
 		Expect(inspectSrc.ExitCode()).To(Equal(0))
 		Expect(inspectSrc.OutputToString()).To(Equal("/tmp"))
 
-		inspectDestination := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Destination}}"})
+		inspectDestination := podmanTest.Podman([]string{"inspect", "test", "--format", "{{(index .Mounts 0).Destination}}"})
 		inspectDestination.WaitWithDefaultTimeout()
 		Expect(inspectDestination.ExitCode()).To(Equal(0))
 		Expect(inspectDestination.OutputToString()).To(Equal("/test1"))
 
-		inspectDst := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Dst}}"})
```

Comment:
might be good to keep one or two of the --latest tests

Reply:
There are still plenty of tests that use --latest, and maybe we should setup one test for each command to try --latest, but we should not use it when testing other commands.